package/scripts/auth.py

```diff
@@ -13,7 +13,6 @@
 # the License.
 #
 
-import sys
 import ambari_helpers as helpers
 from resource_management import *
```

Comment:
Agreed. This is copied from the Ambari code. The module is *really* ugly and it would require a complete refactor to change every instance of an Ambari-supplied object. Lint checking is complaining about this, but I don't have a good solution, at this time.

https://github.com/apache/ambari/blob/trunk/ambari-server/src/main/resources/common-services/HDFS/2.1.0.2.0/package/scripts/hdfs.py#L22